### PR TITLE
Microbenchmark for encoding+decoding single-op GraphProto and ModelProto

### PR DESCRIPTION
**Pull request by Maratyszcza**
This PR is in relation to decision about proper message for `onnxGetBackendCompatibility` in #551
In my experiments `GraphProto` version takes `19 us`, `ModelProto` version - `20 us`.

### CODE REVIEW

**Comment by yinghai** [comment]
@onnxbot restest this please

**Comment by yinghai** [comment]
@onnxbot retest this please

File: bench/protobuf.cc [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+  onnx::TensorShapeProto* shape = tensor_type->mutable_shape();
+  shape->add_dim()->set_dim_value(n);
+  shape->add_dim()->set_dim_value(c);
+  shape->add_dim()->set_dim_value(h);
+  shape->add_dim()->set_dim_value(w);
+}
+
+inline void createValueInfo2D(
+    onnx::ValueInfoProto& value_info,
```
Comment (yinghai) [inline bench/protobuf.cc]: Please change `onnx::` to `ONNX_NAMESPACE::`. Or use `using`.

Comment (Maratyszcza) [reply bench/protobuf.cc]: Fixed

File: bench/protobuf.cc [hunk 2]
```diff
@@ -0,0 +1,146 @@
+#include <benchmark/benchmark.h>
+
```
Comment (linkerzhang) [inline bench/protobuf.cc]: let's put this into \tools folder please. Sounds good?

Comment (Maratyszcza) [reply bench/protobuf.cc]: Done! Clear to merge?

**Pull request merged**